api/src/main/java/uk/gov/dvsa/mot/vehicle/read/core/VehicleReadService.java

```diff
@@ -14,6 +15,8 @@
 
     List<Vehicle> findByRegistration(String registration);
 
+    List<DvlaVehicle> findDvlaVehicleByRegistration(String registration);
```

Comment:
Yes @Thomas-Franklin, we just want two, for persistence layer and for JSON response. :P